### PR TITLE
Implement "all namespaces" mode

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Implement "all namespaces" mode

## Referenced issue:

https://github.com/redhat-appstudio/pvc-cleaner/issues/2

### Small demo:

https://www.youtube.com/watch?v=XV7rOsgzz2w

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
Pr updated.

**Comment by AndrienkoAleksandr** [comment]
@Michkov I handled feedback, pr updated one more time.

File: pkg/pod_cleaner/cmd/pod_cleaner.go [hunk 1]
```diff
@@ -87,10 +92,13 @@ func main() {
 	// Remove pvc subfolders in parallel.
```
Comment (Michkov) [inline pkg/pod_cleaner/cmd/pod_cleaner.go:92]: `if "pv-"+pipelinerun.ObjectMeta.Name == pvcSubPath.Name()` -> this not usual subpath folder, seems that it's only in `m2-builds.sh`, in real world it's like this -> https://github.com/redhat-appstudio/build-definitions/blob/main/.tekton/pull-request.yaml#L54

Comment (Michkov) [reply pkg/pod_cleaner/cmd/pod_cleaner.go:92]: And appstudio is using subfolders:
```
sh-4.4$ ls /test
devfile-sample-code-with-quarkus  devfile-sample-java-springboot-basic  devfile-sample-python-basic  lost+found
sh-4.4$ ls /test/devfile-sample-code-with-quarkus/                                  
initialbuild-2022-Feb-11_06-56-04  initialbuild-2022-Feb-11_08-48-10
```

**Review by mmorhun — APPROVED** [review]
LGTM

File: pkg/pod_cleaner/cmd/pod_cleaner.go (hunk 1 above)
Comment (AndrienkoAleksandr) [reply pkg/pod_cleaner/cmd/pod_cleaner.go]: I changed implementation. PVC cleaner will use pvc subfolders from pipelinerun k8s resource.

**Review by Michkov — APPROVED** [review]

**Pull request merged**